examples/routing_service/routing_service_msql_adapter/cpp/ConnectorMsqlDb.hpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * (c) 2020 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+ *
+ * RTI grants Licensee a license to use, modify, compile, and create derivative
+ * works of the Software.  Licensee has the right to distribute object form
+ * only for use with RTI products.  The Software is provided "as is", with no
+ * warranty of any type, including any warranty for fitness for any purpose.
+ * RTI is under no obligation to maintain or support the Software.  RTI shall
+ * not be liable for any incidental or consequential damages arising out of the
+ * use or inability to use the software.
+ */
+
+#ifndef CONNECTOR_MSQLDB_HPP_
+#define CONNECTOR_MSQLDB_HPP_
+
+#include "ConnectorBase.hpp"
+#include <dds/core/xtypes/DynamicData.hpp>
+#include <rti/routing/PropertySet.hpp>
+#include <sqltypes.h>
+#include <windows.h>
+
+/*
+ * MSQL implementation of a connector
+*/
+class ConnectorMsqlDb : public ConnectorBase {
+public:
+    ConnectorMsqlDb(
+            const std::string topicName,
```

Comment:
This should be a reference to const std::string

Reply:
Done